plugins/server.go

```diff
@@ -68,6 +68,18 @@ func (s *Server) start() error {
 	if err != nil {
 		return fmt.Errorf("error getting environment configuration: %w", err)
 	}
+
+	err = loop.SetupTracing(loop.TracingConfig{
+		Enabled:         envCfg.TracingEnabled(),
+		CollectorTarget: envCfg.TracingCollectorTarget(),
+		NodeAttributes:  envCfg.TracingAttributes(),
+		SamplingRatio:   envCfg.TracingSamplingRatio(),
+	})
+	if err != nil {
+		// non blocking to server start
+		s.lggr.Errorf("Failed to setup tracing: %s", err)
```

Comment:
Oh this should be `Logger`. `lggr` is the field of an embedded type. I made this mistake before. My PR set changes it so it can't happen again.
```suggestion
		s.Logger.Errorf("Failed to setup tracing: %s", err)
```

Reply:
Ah, good catch. Updated